docs/fleet/provision.md

```diff
@@ -1,278 +1,187 @@
 
-{{% /tab %}}
-{{% tab name="Install using a configuration file" %}}
-
-If you want to install the Viam Agent on a machine that you have not yet created in the Viam app, follow these steps:
-
-1. Create a configuration file with the desired configuration for your machine. You can:
-
-   - You can [create a new machine in the Viam app](/cloud/machines/#add-a-new-machine) and configure it as desired, then switch to **Raw JSON** mode and copy the configuration shown into a new file on your machine.
-   - You can base your configuration on the [example configuration file](/internals/local-configuration-file/#example-json-configuration-file), and adjust as needed.
-   - You can use an existing configuration file from a deployed machine, or adapt such a file as needed to fit the specifications of your new machine.
-
-1. Place this file in the following location on the machine you wish to install the Viam Agent to: <file>/etc/viam.json</file>
-
-1. Run the following command to install the Viam Agent on your machine:
-
-   ```sh {class="command-line" data-prompt="$"}
-   sudo /bin/sh -c "$(curl -fsSL https://storage.googleapis.com/packages.viam.com/apps/viam-agent/install.sh)"
-   ```
+Using `agent-provisioning`, this company can ship their machines directly to customers with `viam-agent` installed.
+When a customer sets up their machine, the provisioning subsystem installs `viam-server`.
+By having the end customer set up the machine, the company:
 
-{{% /tab %}}
-{{< /tabs >}}
+- eliminates per-device setup and individualization at the factory
+- allows for tailored configurations per customer as needed
+- allows customer to provide their own WiFi credentials
 
-### Update the Viam Agent and `viam-server`
+{{< alert title="Support Notice" color="note" >}}
 
-The Viam Agent automatically updates both itself and `viam-server` as new updates are released.
-You can also configure update behavior for the Agent and `viam-server` using the [Viam app](https://app.viam.com/).
+Provisioning is supported and tested only on Debian 11 (Bullseye), and 12 (Bookworm) but should work on most distros using NetworkManager v1.42 (or newer) as well.
+For Bullseye, the installation of `viam-agent` changes the network configuration to use NetworkManager.
 
-{{< alert title="Important" color="note" >}}
-When the Viam Agent updates either itself or `viam-server`, you must restart these services in order to use the new version.
-When you stop or restart the Viam Agent, the agent will stop or restart `viam-server` as well.
 {{< /alert >}}
 
-### Manage the Viam Agent
+## End user experience
 
-The Viam Agent is installed as a `systemd` service named `viam-agent`.
+End users receive a machine, and use either a captive web portal or mobile app to complete the machine setup.
```

Comment:
People using the typescript sdk... would their web apps replace the web portal and be run from the machine?

Reply:
I think we cleared this up in slack, but for posterity. No, and that's the problem. We currently don't have a way to customize the browser-based experience in the web portal. Thus provisioning apps (regardless of SDK they use) need to be self-contained mobile apps (app store style) for the time being.